src/Model/AbstractModel.php

```diff
@@ -275,10 +275,14 @@ public function getPackagedName($namespaced = false)
         if ($namespaced && $this->getNamespace() !== '') {
             $nameParts[] = sprintf('\%s\\', $this->getNamespace());
         }
+        $cleanName = $this->getCleanName();
         if ($this->getGenerator()->getOptionPrefix() !== '') {
             $nameParts[] = $this->getGenerator()->getOptionPrefix();
+        } else {
+            $cleanName = self::replaceReservedPhpKeyword($cleanName, null);
```

Comment:
If null Arguments trigger a special handling I like them to be called explicitly. I think that makes the code more readable.